docs/4.x/generated/Piwik/API/Request.md

```diff
@@ -120,11 +120,11 @@ object.
 #### Signature
 
 
-- *Returns:*  [`DataTable`](../../Piwik/DataTable.md)|`Piwik\API\Map`|`string` &mdash;
+- *Returns:*  `Piwik\API\DataTable`|`Piwik\API\Map`|`string` &mdash;
```

Comment:
fyi @mattmary these class names are not generated correctly. It should be `Piwik\DataTable` and it should link to the correct page